modules/tensor_mechanics/doc/content/source/materials/abaqus/AbaqusUMATStress.md

```diff
@@ -49,6 +49,17 @@ and `NPT` (integration point number), while MOOSE uses 0-based indexing
 internally. We therefore add one to the element and integration point numbers
 from MOOSE before passing them to the UMAT.
 
+## Small and total strain formulations
+
+If material properties provided by an incremental strain formulation do not
+exist they will be filled with NaN (not a number) values. If in this case the
```

Comment:
If they don't exist: Aren't we erroring out in `AbaqusUMATStress::initialSetup`?

Reply:
Yeah, the minute I walked out of the pod I said "$#!+", because I remembered I forgot to update the docs...